javascript/cra-starter/README.md

```diff
@@ -1 +1,44 @@
-# Basic CRA Starter
\ No newline at end of file
+# thirdweb CRA Starter
+
+A starter thirdweb template for a React application.
+
+### Installation
+
+Clone this project locally using [degit](https://npmjs.org/package/degit):
+
+```bash
+npx degit https://github.com/thirdweb-dev/examples/tree/main/javascript/cra-starter
+```
+
+Then install the dependencies:
+
+```
+npm install
+```
+
+
+### Local Development
+
+
+
+Run the following command to run the app locally:
+
+```
+npm run dev
+```
+
+
+### Building for production
+
+
+
```

Comment:
there's triple spaces in a multitude of places, any reason for it? one space is enough here